mkdocs.yml

```diff
@@ -96,7 +96,7 @@ plugins:
         paths: [
           # from nav_repos above
           "./temp_dir/core/src/",
-          "./temp_dir/workflow/src/",
+          # "./temp_dir/workflow/src/",
```

Comment:
... and here!
```suggestion
```

Reply:
Also check out #586 – that covers the rest of the updates you'll need to get the workflow docs integrated